src/rpc/coinjoin.cpp

```diff
@@ -49,7 +51,8 @@ static UniValue coinjoin(const JSONRPCRequest& request)
         }
     }
 
-    auto it = coinJoinClientManagers.find(wallet->GetName());
+    auto clientman = ::coinJoinClientManagers->Get(*wallet);
```

Comment:
`clientman` isn't a great name `cjclientman` would be better

Reply:
Non-member definitions have been renamed to `cj_clientman`. Resolved in latest push.